espnet2/tasks/asr.py

```diff
@@ -74,6 +76,15 @@
     default="utterance_mvn",
     optional=True,
 )
+preencoder_choices = ClassChoices(
+    name="preencoder",
+    classes=dict(
+        sinc=LightweightSincConvs,
+    ),
+    type_check=AbsFrontend,
```

Comment:
This is not AbsFrontend.
Please define AbsPreEncoder

Reply:
OK